examples/deployments/docker/docker-compose.yaml

```diff
@@ -48,6 +47,7 @@ services:
       - "9300:9300"
     volumes:
       - elasticdata:/usr/share/elasticsearch/data/
+
   kibana:
```

Comment:
@frascuchon is kibana really needed? Can we remove it?

Reply:
I would vote for removing it yes

Reply:
or at least comment it out, maybe better